README.md

```diff
@@ -81,5 +81,4 @@ For each sign and metric, either the first match is incorrect, or there is a mor
       [^4]: Jack Hessel, Ari Holtzman, Maxwell Forbes, Ronan Le Bras, and Yejin Choi.
 2021. [CLIPScore: A Reference-free Evaluation Metric for Image Captioning](https://aclanthology.org/2021.emnlp-main.595/).
       In Proceedings of the 2021 Conference on Empirical Methods in Natural Language Processing, pages 7514–7528, Online
-      and
-      Punta Cana, Dominican Republic. Association for Computational Linguistics.
+      and Punta Cana, Dominican Republic. Association for Computational Linguistics.
```

Comment:
missing new line at the end of the file

Reply:
i guess i'm wrong then, github doesn't show it, but if the linter doesn't complain, all good